### ISSUE TITLE
API Viewer goes blank after git add

### ISSUE THREAD

**Issue by jemisonf** [42Crunch/vscode-openapi#22]
Steps to reproduce:
1. Have an openapi yaml file open in an active git repository
2. Make a change to the file
3. `git add` the yaml file
4. The API previewer sidebar panel will go blank. Closing and re-opening the file will fix the issue.

Other installed extensions:
```
42crunch.vscode-openapi-1.8.13
alireza94.theme-gotham-0.4.0
arjun.swagger-viewer-2.2.2
derek-nextty.openapi-viewer-0.1.9
eamodio.gitlens-10.0.1
ms-python.python-2019.9.34911
philosowaffle.openapi-designer-0.3.0
redhat.vscode-yaml-0.5.3
vscodevim.vim-1.10.2
vscodevim.vim-1.11.0
```

I can test this on a blank install when I get a chance.

**Comment by ak1394** [comment]
Thanks for reporting! I'll investigate.

**Comment by jemisonf** [comment]
Awesome, thank you. I can provide more details about my specific setup if it doesn't reproduce on a blank install.

**Comment by ak1394** [comment]
Issue is resolved and fix will be published in the next 1.8.19 update.

**Comment by jemisonf** [comment]
Great news, thanks @ak1394!
